src/main/kotlin/org/jitsi/jicofo/conference/colibri/ColibriAllocation.kt

```diff
@@ -14,5 +14,7 @@ data class ColibriAllocation(
     /** The region of the bridge */
     val region: String?,
     /** An identifier for the session */
-    val bridgeSessionId: String?
+    val bridgeSessionId: String?,
+    /** The SCTP port (used for both local and remote port) advertised by the bridge, if any */
+    val sctpPort: Int?
```

Comment:
Would it be more 'obvious' to put the default port here rather than in `ParticipantInviteRunnable`?

Reply:
I don't want to set the default port here, because in the future when there's only colibri2 there will be no default port and `ColibriAllocation.sctpPort` will control whether SCTP is signaled to the client or not.

I removed the double definition, I think it's a bit less confusing now